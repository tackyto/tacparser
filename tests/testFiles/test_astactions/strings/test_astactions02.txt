"Test01" {
    "FamilyName01" {
        {name = "Aaaa Aaaa", age = 41, sex = Man, phone = 0123-4567-8901 }
        {name = "Aaaa Bbbb", age = 42, sex = Female, phone = 0123-4567-8902 }
        {name = "Aaaa Cccc", age = 15, sex = M}
        {name = "Aaaa Dddd", age = 12, sex = F}
    }
    "FamilyName02" {
        {name = "Bbbb Aaaa", age = 45, sex = Man, phone = 0123-4567-8901 }
        {name = "Bbbb Bbbb", age = 44, sex = Female }
        {name = "Bbbb Cccc", age = 13, sex = M}
        {name = "Bbbb Dddd", age = 11, sex = F}
    }
}
