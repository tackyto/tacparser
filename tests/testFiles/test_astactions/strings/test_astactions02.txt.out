ASTActionsTest : (1, 0 - 14, 2 : 0 - 541) : {}
    Society : (1, 0 - 14, 2 : 0 - 541) : {}
        SocietyName : (1, 0 - 1, 9 : 0 - 9) : {}
            Literal : (1, 0 - 1, 9 : 0 - 9) : {}
                @Tarminal : (1, 1 - 1, 7 : 1 - 7) "Test01"
        Family : (2, 4 - 8, 4 : 15 - 291) : {}
            FamilyName : (2, 4 - 2, 19 : 15 - 30) : {}
                Literal : (2, 4 - 2, 19 : 15 - 30) : {}
                    @Tarminal : (2, 5 - 2, 17 : 16 - 28) "FamilyName01"
            Person : (3, 9 - 3, 73 : 41 - 105) : {}
                Name : (3, 9 - 3, 29 : 41 - 61) : {}
                    Literal : (3, 16 - 3, 27 : 48 - 59) : {}
                        @Tarminal : (3, 17 - 3, 26 : 49 - 58) "Aaaa Aaaa"
                Age : (3, 29 - 3, 39 : 61 - 71) : {}
                    Number : (3, 35 - 3, 37 : 67 - 69) : {}
                        @Tarminal : (3, 35 - 3, 37 : 67 - 69) "41"
                Sex : (3, 39 - 3, 50 : 71 - 82) : {}
                    MorF : (3, 45 - 3, 48 : 77 - 80) : {}
                        Male : (3, 45 - 3, 48 : 77 - 80) : {}
                Phone : (3, 50 - 3, 73 : 82 - 105) : {}
                    NumberHyphen : (3, 58 - 3, 73 : 90 - 105) : {}
                        @Tarminal : (3, 58 - 3, 63 : 90 - 95) "0123-"
                        @Tarminal : (3, 63 - 3, 68 : 95 - 100) "4567-"
                        @Tarminal : (3, 68 - 3, 72 : 100 - 104) "8901"
            Person : (4, 9 - 4, 76 : 116 - 183) : {}
                Name : (4, 9 - 4, 29 : 116 - 136) : {}
                    Literal : (4, 16 - 4, 27 : 123 - 134) : {}
                        @Tarminal : (4, 17 - 4, 26 : 124 - 133) "Aaaa Bbbb"
                Age : (4, 29 - 4, 39 : 136 - 146) : {}
                    Number : (4, 35 - 4, 37 : 142 - 144) : {}
                        @Tarminal : (4, 35 - 4, 37 : 142 - 144) "42"
                Sex : (4, 39 - 4, 53 : 146 - 160) : {}
                    MorF : (4, 45 - 4, 51 : 152 - 158) : {}
                        Female : (4, 45 - 4, 51 : 152 - 158) : {}
                Phone : (4, 53 - 4, 76 : 160 - 183) : {}
                    NumberHyphen : (4, 61 - 4, 76 : 168 - 183) : {}
                        @Tarminal : (4, 61 - 4, 66 : 168 - 173) "0123-"
                        @Tarminal : (4, 66 - 4, 71 : 173 - 178) "4567-"
                        @Tarminal : (4, 71 - 4, 75 : 178 - 182) "8902"
            Person : (5, 9 - 5, 46 : 194 - 231) : {}
                Name : (5, 9 - 5, 29 : 194 - 214) : {}
                    Literal : (5, 16 - 5, 27 : 201 - 212) : {}
                        @Tarminal : (5, 17 - 5, 26 : 202 - 211) "Aaaa Cccc"
                Age : (5, 29 - 5, 39 : 214 - 224) : {}
                    Number : (5, 35 - 5, 37 : 220 - 222) : {}
                        @Tarminal : (5, 35 - 5, 37 : 220 - 222) "15"
                Sex : (5, 39 - 5, 46 : 224 - 231) : {}
                    MorF : (5, 45 - 5, 46 : 230 - 231) : {}
                        Male : (5, 45 - 5, 46 : 230 - 231) : {}
            Person : (6, 9 - 6, 46 : 242 - 279) : {}
                Name : (6, 9 - 6, 29 : 242 - 262) : {}
                    Literal : (6, 16 - 6, 27 : 249 - 260) : {}
                        @Tarminal : (6, 17 - 6, 26 : 250 - 259) "Aaaa Dddd"
                Age : (6, 29 - 6, 39 : 262 - 272) : {}
                    Number : (6, 35 - 6, 37 : 268 - 270) : {}
                        @Tarminal : (6, 35 - 6, 37 : 268 - 270) "12"
                Sex : (6, 39 - 6, 46 : 272 - 279) : {}
                    MorF : (6, 45 - 6, 46 : 278 - 279) : {}
                        Female : (6, 45 - 6, 46 : 278 - 279) : {}
        Family : (8, 4 - 14, 0 : 291 - 539) : {}
            FamilyName : (8, 4 - 8, 19 : 291 - 306) : {}
                Literal : (8, 4 - 8, 19 : 291 - 306) : {}
                    @Tarminal : (8, 5 - 8, 17 : 292 - 304) "FamilyName02"
            Person : (9, 9 - 9, 73 : 317 - 381) : {}
                Name : (9, 9 - 9, 29 : 317 - 337) : {}
                    Literal : (9, 16 - 9, 27 : 324 - 335) : {}
                        @Tarminal : (9, 17 - 9, 26 : 325 - 334) "Bbbb Aaaa"
                Age : (9, 29 - 9, 39 : 337 - 347) : {}
                    Number : (9, 35 - 9, 37 : 343 - 345) : {}
                        @Tarminal : (9, 35 - 9, 37 : 343 - 345) "45"
                Sex : (9, 39 - 9, 50 : 347 - 358) : {}
                    MorF : (9, 45 - 9, 48 : 353 - 356) : {}
                        Male : (9, 45 - 9, 48 : 353 - 356) : {}
                Phone : (9, 50 - 9, 73 : 358 - 381) : {}
                    NumberHyphen : (9, 58 - 9, 73 : 366 - 381) : {}
                        @Tarminal : (9, 58 - 9, 63 : 366 - 371) "0123-"
                        @Tarminal : (9, 63 - 9, 68 : 371 - 376) "4567-"
                        @Tarminal : (9, 68 - 9, 72 : 376 - 380) "8901"
            Person : (10, 9 - 10, 52 : 392 - 435) : {}
                Name : (10, 9 - 10, 29 : 392 - 412) : {}
                    Literal : (10, 16 - 10, 27 : 399 - 410) : {}
                        @Tarminal : (10, 17 - 10, 26 : 400 - 409) "Bbbb Bbbb"
                Age : (10, 29 - 10, 39 : 412 - 422) : {}
                    Number : (10, 35 - 10, 37 : 418 - 420) : {}
                        @Tarminal : (10, 35 - 10, 37 : 418 - 420) "44"
                Sex : (10, 39 - 10, 52 : 422 - 435) : {}
                    MorF : (10, 45 - 10, 52 : 428 - 435) : {}
                        Female : (10, 45 - 10, 52 : 428 - 435) : {}
            Person : (11, 9 - 11, 46 : 446 - 483) : {}
                Name : (11, 9 - 11, 29 : 446 - 466) : {}
                    Literal : (11, 16 - 11, 27 : 453 - 464) : {}
                        @Tarminal : (11, 17 - 11, 26 : 454 - 463) "Bbbb Cccc"
                Age : (11, 29 - 11, 39 : 466 - 476) : {}
                    Number : (11, 35 - 11, 37 : 472 - 474) : {}
                        @Tarminal : (11, 35 - 11, 37 : 472 - 474) "13"
                Sex : (11, 39 - 11, 46 : 476 - 483) : {}
                    MorF : (11, 45 - 11, 46 : 482 - 483) : {}
                        Male : (11, 45 - 11, 46 : 482 - 483) : {}
            Person : (12, 9 - 12, 46 : 494 - 531) : {}
                Name : (12, 9 - 12, 29 : 494 - 514) : {}
                    Literal : (12, 16 - 12, 27 : 501 - 512) : {}
                        @Tarminal : (12, 17 - 12, 26 : 502 - 511) "Bbbb Dddd"
                Age : (12, 29 - 12, 39 : 514 - 524) : {}
                    Number : (12, 35 - 12, 37 : 520 - 522) : {}
                        @Tarminal : (12, 35 - 12, 37 : 520 - 522) "11"
                Sex : (12, 39 - 12, 46 : 524 - 531) : {}
                    MorF : (12, 45 - 12, 46 : 530 - 531) : {}
                        Female : (12, 45 - 12, 46 : 530 - 531) : {}
    _EOF : (14, 2 - 14, 2 : 541 - 541) : {}
